pkg/render/manager.go

```diff
@@ -534,6 +534,7 @@ func (c *managerComponent) voltronContainer() corev1.Container {
 
 	if c.cfg.Tenant != nil {
 		env = append(env, corev1.EnvVar{Name: "VOLTRON_TENANT_NAMESPACE", Value: c.cfg.Tenant.Namespace})
+		env = append(env, corev1.EnvVar{Name: "VOLTRON_LINSEED_ENDPOINT", Value: fmt.Sprintf("https://tigera-linseed.%s.svc.cluster.local", c.cfg.Tenant.Namespace)})
```

Comment:
".cluster.local" isn't always the correct suffix. I think we should stop it at `.svc`